app/grandchallenge/reader_studies/views.py

```diff
@@ -1427,24 +1408,11 @@ def new_interface_url(self):
             kwargs={"slug": self.base_object.slug},
         )
 
-    def process_data_for_object(self, data):
-        """Creates a display set"""
-        non_civ_data_keys = self.form_class.Meta.non_civ_fields
-
-        instance = DisplaySet.objects.create(
-            reader_study=self.base_object,
-            **{k: data.get(k) for k in non_civ_data_keys},
+    def create_object(self, non_civ_data):
+        return DisplaySet.objects.create(
+            reader_study=self.base_object, **non_civ_data
         )
```

Comment:
This is not used anywhere now, is it?

Reply:
Quite right, that other instance should really have made me look for the other one! Thanks!